test/failure_handler/src/share/conf/windows.properties

```diff
@@ -57,7 +57,7 @@ native.stack.args=-c "~*kP n;qd" -p %p
 native.stack.params.repeat=6
 
 native.core.app=cdb
-native.core.args=-c ".dump /f core.%p;qd" -p %p
+native.core.args=-c ".dump /ma core.%p;qd" -p %p
 native.core.params.timeout=3600000
```

Comment:
Hello Daniel, I found it surprising that this takes 2 hours to complete. The failure handler infrastructure has timeout handling built in, after which it kills the failure handler action (the process). Looking at the value specified here it translates to a timeout of 60 minutes (which is too high by the way). So I looked around in some other files and I think there might be a bug here. In other files (linux.properties and mac.properties), I notice the timeout is specified as:

```
native.core.timeout=600000
```
Notice the absence of "params" part in that key. I wonder if that is playing a role here and whether we should fix this key. While at it, perhaps we should also reduce this timeout to may be something lesser (1 hour seems to high). Linux and macosx use a value of `600000` which is 10 minutes. If Windows requires a few more minutes then that's understandable but perhaps we should set it to a maximum of 30 minutes maybe?

Reply:
> Notice the absence of "params" part in that key. I wonder if that is playing a role here and whether we should fix this key.

Actually ignore that part. I had a look at the internal logs that you referenced. It appears that this form of specifying the timeout (through the use of `params` key) seems to work too. The reason why it took 2 hours is because it ran that command against 2 separate processes and each one timed out after one hour:

```
[2023-11-23 21:45:40] [cdb.exe, -c, ".dump, /f, core.12345;qd", -p, 12345] timeout=3600000
...
0:001> WARNING: tool timed out: killed process after 3600000 ms
----------------------------------------
[2023-11-23 22:45:40] exit code: -2 time: 3600006 ms
----------------------------------------


----------------------------------------
[2023-11-23 22:47:36] [cdb.exe, -c, ".dump, /f, core.6789;qd", -p, 6789] timeout=3600000
----------------------------------------

...
0:063> WARNING: tool timed out: killed process after 3600000 ms
----------------------------------------
[2023-11-23 23:47:36] exit code: -2 time: 3599996 ms
----------------------------------------

```

Reply:
Out of curiosity, do we know why it takes more than an hour? Is it attempting to connect to the Microsoft symbol server?

Reply:
It's ignoring the rest of the command line after `/f`, which means it ignores the `;qd` (quit and detach) part and enters the interactive mode.

Reply:
> It's ignoring the rest of the command line after `/f`, which means it ignores the `;qd` (quit and detach) part and enters the interactive mode.

Wonderful :-)    Does switching to `/mA` change that?

Reply:
Yes, with `/mA` the command exits as soon as the dump is completed.
I couldn't find a way to make `cdb` not enter interactive mode on unrecognized parameter, so I left that part as is.

Reply:
> Yes, with `/mA` the command exits as soon as the dump is completed. I couldn't find a way to make `cdb` not enter interactive mode on unrecognized parameter, so I left that part as is.

Okay, maybe it can be figure out another time.